release/src/version-helpers.ts

```diff
@@ -239,16 +245,22 @@ export async function getLastReleaseTag({
 }
 
 export const findNextPatchVersion = (version: string) => {
-  if (!isValidVersionString(version) || isRCVersion(version)) {
+  if (!isValidVersionString(version)) {
     throw new Error(`Invalid version string: ${version}`);
   }
 
-  const [major, minor, patch] = version
+  const [mainVersion, suffix] = version.split("-");
+
+  const [major, minor, patch] = mainVersion
     .replace(/(v1|v0)\./, "")
     .split(".")
     .map(Number);
 
-  return `v0.${major}.${minor ?? 0}.${(patch ?? 0) + 1}`;
+  const baseVersion = `v0.${major}.${minor ?? 0}.${(patch ?? 0) + 1}`;
```

Comment:
Since `NaN ?? 0` is NaN, could the string "NaN" end up appearing in `baseVersion`?

Reply:
with the current `isValidVersionString()` check, it's impossible to get a `NaN` minor or patch, and there's unit test coverage for it. Nonetheless, I've updated it to ` || 0` to make it more robust in case the shape of a valid version changes.